app/views/layouts/application.html.slim

```diff
@@ -2,7 +2,10 @@ doctype html
 html
   head
     title Portus
-    = stylesheet_link_tag 'application', media: 'all'
+    - if Rails.application.assets.find_asset(params[:controller])
+      = stylesheet_link_tag params[:controller], media: 'all'
+    - else
+      = stylesheet_link_tag 'application', media: 'all'
```

Comment:
I understand that this way is safer, but what if we just assumed that there was *always* a controller asset ? This would even make this simpler and it would remove an `if` statement on such a heavily used page. Yes, we would be quite explicit on controllers with no style whatsoever, but I think that it would be better for code clarity actually. What do you think ?